Update troubleshooting guide for headless services (#255)
submariner-io/submariner-website: src/content/troubleshooting/_index.en.md

Events:  <none>
```

> In case of a HeadlessService, `Ips` field in will contain list of Endpoint IPs for that service instead of `ServiceIP`.

Reviewer: ```suggestion
In case of a headless Service, `Ips` field will contain a list of Endpoint IPs for that service instead of `ServiceIP`.
```

Reviewer 2: ```suggestion
Note that for a headless Service, the `Ips` field will contain the list of associated Endpoint IPs.
```

If the data is not correct, you can manually edit the ServiceImport resource to set the correct IP as a workaround and create an
[issue](https://github.com/submariner-io/lighthouse/issues) with relevant information.

> If the `Ips` is correct in ServiceImport resource but still not showing up in DNS query, check connectivity to the cluster
> using [`subctl show endpoint`](../deployment/subctl/_index.en.md#show-endpoints). Service Discovery DNS only returns IPs from connected clusters.

Reviewer 2: ```suggestion
If the ServiceImport `Ips` are correct but still not being returned from DNS queries, check the connectivity to the cluster
using [`subctl show endpoint`](../deployment/subctl/_index.en.md#show-endpoints). The Lighthouse CoreDNS Server only returns IPs from connected clusters.
```
